Makefile

```diff
@@ -232,9 +232,7 @@ install.cni:
 install.docker: docker-docs
 	install ${SELINUXOPT} -D -m 755 docker $(BINDIR)/docker
 	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man1
-	install ${SELINUXOPT} -d -m 755 $(MANDIR)/man5
```

Comment:
Do you really want to remove this?

Reply:
Yes, their are no man5 man pages.

Reply:
As @vrothberg pointed out, we're currently shipping libpod.conf.5.  Do we want to stop shipping that?

Reply:
No.  That is not installed here, it is installed in the standard install.  This is just about installing links for Docker alias, so people accidently typing docker commands or man docker build, will get information back.